ts/screens/authentication/UnlockAccessScreen.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Route, useRoute } from "@react-navigation/native";
+import UnlockAccessComponent, {
+  UnlockAccessProps
+} from "./UnlockAccessComponent";
+
+const UnlockAccessScreen = () => {
+  const route = useRoute<Route<"UNLOCK_ACCESS_SCREEN", UnlockAccessProps>>();
```

Comment:
Instead of manually type "UNLOCK_ACCESS_SCREEN " you can derive it from `ROUTES.UNLOCK_ACCESS_SCREEN `, in this way if, for any reason, its value changes no action will be required.

Reply:
good suggestion! I applied it! 😁